provider/resource_instance.go

```diff
@@ -23,11 +23,23 @@ func resourceInstance() *schema.Resource {
 			StateContext: schema.ImportStatePassthroughContext,
 		},
 		Schema: map[string]*schema.Schema{
-			"name": {
+			"resource_id": {
```

Comment:
Cannot use `id` in the provider, the `id` keyword is limited and users cannot set it manually.